data/meson.build

```diff
@@ -0,0 +1,4 @@
+# Copyright 2018 Endless Mobile, Inc.
+
+subdir('css')
```

Comment:
Unlike with Automake, you don't need this intermediate step. You can just do `subdir('data/css')` in the toplevel `meson.build`.